torchmd/forces.py

```diff
@@ -38,15 +38,10 @@ def __init__(
     ):
         self.par = parameters
         if terms is None:
-            terms = (
-                "electrostatics",
-                "lj",
-                "bonds",
-                "angles",
-                "dihedrals",
-                "1-4",
-                "impropers",
+            raise RuntimeError(
+                f'Set force terms or leave empyt brackets [].\nAvaliable options: "bonds", "angles", "dihedrals", "impropers", "1-4", "electrostatics", "lj", "repulsion", "repulsioncg".'
```

Comment:
empyt = empty
Avaliable = Available

Reply:
It's probably a right moment to say that I'm dyslexic